src/components/DatePicker/components/Weekday/Weekday.tsx

```diff
@@ -21,7 +20,7 @@ export const Weekday = memo(function Weekday({
   );
 
   return (
-    <div aria-label={WeekdayEnum[label]} className={className}>
```

Comment:
So uh, it turns out here we were just using the enum key name (which would always be english) instead of actually translating weekday name.

I've updated this so we pass in the translated key into the `label` prop from Month.tsx.

Reply:
Worth mentioning in the release notes?